src/pages/en/guides/typescript.md

```diff
@@ -103,6 +103,12 @@ const { greeting = 'Hello', name } = Astro.props;
 <h2>{greeting}, {name}!</h2>
 ```
 
+:::tip
+
+To type a component that denies props, you can pass `type Props = Record<string, never>;` for zero props and `type Props = { children: any; };` for no props except for the default slot.
+
+:::
```

Comment:
I wonder if this is important enough to not be a `tip`. Maybe we could collect these different scenarios like this:

```suggestion
### Common prop type patterns

- If your component takes no props or slotted content, you can use `type Props = Record<string, never>`.

- If your component must be passed children to its default slot, you can enforce this by using `type Props = { children: any; };`.
```

Reply:
that's probably better